widget/playground/src/utils/export.ts

```diff
@@ -0,0 +1,124 @@
+import type { WidgetConfig } from '@rango-dev/widget-embedded';
+
+import stringifyObject from 'stringify-object';
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
```

Comment:
I skip checking this file changes as it seems to be just moving functions to this file.